### PR TITLE
RTEMIS-131: Modified the Student Application Print.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-131

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Added necessary permission.
- Modified Twig.

### CODE REVIEW

File: modules/rte_mis_student/rte_mis_student.module [hunk 1]
```diff
@@ -33,6 +36,112 @@ function rte_mis_student_theme() {
...
+                // Get the school udise code from the term Id,
+                // provided by school details.
+                $school_udise_id = $school_details->get('field_udise_code')->getString();
+                $school_udise = $term_storage->loadByProperties([
+                  'tid' => $school_udise_id,
+                  'vid' => 'school',
+                  'status' => 1,
+                  'field_workflow' => 'school_workflow_approved',
+                ]);
+                $school_udise_code = reset($school_udise)->getName();
```
Comment (jaswinsingh) [inline modules/rte_mis_student/rte_mis_student.module:99]: Please add safe check here

File: modules/rte_mis_student/rte_mis_student.module [hunk 2]
```diff
@@ -33,6 +36,112 @@ function rte_mis_student_theme() {
...
+
+                // Get the school udise code from the term Id,
+                // provided by school details.
+                $school_udise_id = $school_details->get('field_udise_code')->getString();
+                $school_udise = $term_storage->loadByProperties([
+                  'tid' => $school_udise_id,
+                  'vid' => 'school',
+                  'status' => 1,
+                  'field_workflow' => 'school_workflow_approved',
+                ]);
```
Comment (jaswinsingh) [inline modules/rte_mis_student/rte_mis_student.module:98]: No need to add check here, the school's preference field has all the logic written. We can just load the term and get the name with safe check

File: modules/rte_mis_student/rte_mis_student.module (hunk 1 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_student/rte_mis_student.module]: ok.

File: modules/rte_mis_student/rte_mis_student.module (hunk 2 above)
Comment (RohanSingh-inno) [reply modules/rte_mis_student/rte_mis_student.module]: sure

File: modules/rte_mis_student/rte_mis_student.module [hunk 3]
```diff
@@ -33,6 +36,109 @@ function rte_mis_student_theme() {
...
+    $preferred_school_data = [];
+    $school_info = [];
+    $rte_mis_school_config = \Drupal::config('rte_mis_school.settings');
+    foreach ($preferred_schools as $key => $value) {
+      if (is_numeric($key)) {
+        $school_details = Drupal::entityTypeManager()->getStorage('mini_node')->loadByProperties([
+          'id' => $value['#paragraph']->get('field_school_id')->getString(),
+          'type' => 'school_Details',
+        ]);
+        $school_details = reset($school_details);
```
Comment (jaswinsingh) [inline modules/rte_mis_student/rte_mis_student.module:55]: Same can be done here also

Comment (RohanSingh-inno) [reply modules/rte_mis_student/rte_mis_student.module]: ok

**Review by jaswinsingh — APPROVED** [review]

**Pull request merged**